include/commissioning/commissioner.h

```diff
@@ -50,24 +50,61 @@ extern "C" {
  * This function enables the Thread Commissioner role.
  *
  * @param[in]  aInstance         A pointer to an OpenThread instance.
- * @param[in]  aPSKd             A pointer to the PSKd.
- * @param[in]  aProvisioningUrl  A pointer to the Provisioning URL (may be NULL).
  *
- * @retval kThreadError_None         Successfully started the Commissioner role.
- * @retval kThreadError_InvalidArgs  @p aPSKd or @p aProvisioningUrl is invalid.
+ * @retval kThreadError_None     Successfully started the Commissioner role.
  *
  */
-ThreadError otCommissionerStart(otInstance *aInstance, const char *aPSKd, const char *aProvisioningUrl);
+ThreadError otCommissionerStart(otInstance *aInstance);
 
 /**
  * This function disables the Thread Commissioner role.
  *
- * @param[in]  aInstance  A pointer to an OpenThread instance.
+ * @param[in]  aInstance         A pointer to an OpenThread instance.
+ *
+ * @retval kThreadError_None     Successfully started the Commissioner role.
  *
  */
 ThreadError otCommissionerStop(otInstance *aInstance);
 
 /**
+ * This function adds a Joiner entry.
+ *
+ * @param[in]  aInstance        A pointer to an OpenThread instance.
+ * @param[in]  aExtAddress      A pointer to the Joiner's extended address.
```

Comment:
Shouldn't this parameter be marked as optional, since it looks like you treat a `NULL` as a wildcard.